resources/Dumps/DRMOBS.txt

```diff
@@ -0,0 +1,838 @@
+Microsoft Windows [Version 10.0.19045.2251]
```

Comment:
Please remove this file, it doesn't contain any information that you can't get with the `rrcli` command so it doesn't need to be stored.

If you need to keep these files for reference, please put them under the `tmp/` unless they are useful to be here for everyone.